src/core/Akka.Persistence.Tests/PersistenceConfigSpec.cs

```diff
@@ -4,13 +4,102 @@
+        #region internal classes
+
+        private sealed class TestRequest
+        {
+            public static readonly TestRequest Instance = new TestRequest();
+
+            private TestRequest()
+            {
+            }
+        }
+
+        public class TestJournal : MemoryJournal
+        {
+            private readonly string _testValue;
+
+            public TestJournal(Config config)
+            {
+                _testValue = config.GetString("test-value");
+            }
+
+            protected override bool AroundReceive(Receive receive, object message)
+            {
+                if (message is TestRequest)
+                {
+                    Sender.Tell(_testValue);
+                    return true;
+                }
+                else return base.AroundReceive(receive, message);
+            }
+        }
+
+        public class TestSnapshotStore : LocalSnapshotStore
+        {
+            private readonly string _testValue;
+
+            public TestSnapshotStore(Config config)
+            {
+                _testValue = config.GetString("test-value");
+            }
+
+            protected override bool AroundReceive(Receive receive, object message)
+            {
+                if (message is TestRequest)
+                {
+                    Sender.Tell(_testValue);
+                    return true;
+                }
+                else return base.AroundReceive(receive, message);
+            }
+        }
+
+        #endregion
+
+        private static readonly string SpecConfig = @"
+            akka.persistence.journal {
+                test1 {
+                    class = ""Akka.Persistence.Tests.PersistenceConfigSpec+TestJournal, Akka.Persistence.Tests""
+                    plugin-dispatcher = ""akka.actor.default-dispatcher""
+                    test-value = ""A""
+                }
+                test2 {
+                    class = ""Akka.Persistence.Tests.PersistenceConfigSpec+TestJournal, Akka.Persistence.Tests""
+                    plugin-dispatcher = ""akka.actor.default-dispatcher""
+                    test-value = ""B""
+                }
+            }
+            akka.persistence.snapshot-store {
+                test1 {
+                    class = ""Akka.Persistence.Tests.PersistenceConfigSpec+TestSnapshotStore, Akka.Persistence.Tests""
+                    plugin-dispatcher = ""akka.actor.default-dispatcher""
+                    test-value = ""C""
+                }
+                test2 {
+                    class = ""Akka.Persistence.Tests.PersistenceConfigSpec+TestSnapshotStore, Akka.Persistence.Tests""
+                    plugin-dispatcher = ""akka.actor.default-dispatcher""
+                    test-value = ""D""
+                }
+            }";
+
+        public PersistenceConfigSpec(ITestOutputHelper output = null) : base(SpecConfig, output)
+        {
+        }
+
         [Fact]
         public void Persistence_should_use_inmem_journal_by_default()
```

Comment:
Shouldn't this spec be renamed "Persistence_should_use_**filesys_snapshotstore**_by_default" ?